openshift/template.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: assisted-events-scrape
+objects:
```

Comment:
any chance we can add ``pdb`` as well?
to make sure there's always one instance up during cluster upgrades

Reply:
sure, I'm focusing on deploy it first ASAP, please add a new task for me. 
https://issues.redhat.com/browse/MGMT-7096

Reply:
done